nixos/modules/services/monitoring/cockpit.nix

```diff
@@ -0,0 +1,189 @@
+{ pkgs, config, lib, ... }: let
+  cfg = config.services.cockpit;
+  inherit (lib) types mkEnableOption mkOption mkIf mdDoc literalMD;
+in {
+  options = {
+    services.cockpit = {
+      enable = mkEnableOption (mdDoc "Cockpit");
+      package = mkOption {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Why not [mkPackageOptionMD](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/development/option-declarations.section.md#mkpackageoption-mkpackageoptionmd-sec-option-declarations-util-mkpackageoption)?

Reply:
nice catch